cunumeric/config.py

```diff
@@ -136,33 +139,39 @@ class UnaryOpCode(IntEnum):
     COPY = 7
     COS = 8
     EXP = 9
-    FLOOR = 10
-    INVERT = 11
-    ISINF = 12
-    ISNAN = 13
-    LOG = 14
-    LOGICAL_NOT = 15
-    NEGATIVE = 16
-    SIN = 17
-    SQRT = 18
-    TAN = 19
-    TANH = 20
-    CONJ = 21
-    REAL = 22
-    IMAG = 23
-    GETARG = 24
+    EXP2 = 10
+    FLOOR = 11
+    INVERT = 12
+    ISINF = 13
+    ISNAN = 14
+    LOG = 15
+    LOG10 = 16
+    LOGICAL_NOT = 17
+    NEGATIVE = 18
+    RINT = 19
+    SIGN = 20
+    SIN = 21
+    SQRT = 22
+    TAN = 23
+    TANH = 24
+    CONJ = 25
+    REAL = 26
+    IMAG = 27
+    GETARG = 28
 
 
 @unique
 class UnaryRedCode(IntEnum):
-    MAX = 1
-    MIN = 2
-    PROD = 3
-    SUM = 4
-    ARGMAX = 5
-    ARGMIN = 6
-    CONTAINS = 7
-    COUNT_NONZERO = 8
+    ALL = 1
+    ANY = 2
+    MAX = 3
+    MIN = 4
+    PROD = 5
+    SUM = 6
+    ARGMAX = 7
+    ARGMIN = 8
+    CONTAINS = 9
+    COUNT_NONZERO = 10
 
 
```

Comment:
Since we're editing this file, can you add a note to future developers to match the various enums in this file to the corresponding C++ enums, similar to how it's done for `CuNumericOpCode` above? And add similar back-references from those C++ enums to here?

Reply:
Sure